NickvisionTubeConverter.Shared/Resources/po/pt_PT.po

```diff
@@ -0,0 +1,674 @@
+#, csharp-format
+msgid ""
+"Nicholas Logozzo {0}\n"
+"Fyodor Sobolev {1}\n"
+"DaPigGuy {2}"
+msgstr ""
+"Nicholas Logozzo {0}\n"
+"Fyodor Sobolev {1}\n"
+"DaPigGuy {2}"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/MainWindow.cs:607
+msgid "No"
+msgstr "Não"
+
+#: ../../../Models/DownloadManager.cs:171
+msgid "No downloads running"
+msgstr "Não há transferências a decorrer"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/AddDownloadDialog.cs:213
+msgid "Playlist"
+msgstr "Playlist"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Controls/DownloadRow.cs:148
+msgid "Preparing..."
+msgstr "A preparar..."
+
+#: ../../../../NickvisionTubeConverter.GNOME/Controls/DownloadRow.cs:185
+msgid "Processing..."
+msgstr "A processar..."
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/AddDownloadDialog.cs:391
+#: NickvisionTubeConverter.GNOME/Blueprints/add_download_dialog.blp:122
+msgid "Save Folder"
+msgstr "Pasta de Destino"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/AddDownloadDialog.cs:401
+msgid "Save Folder (Invalid)"
+msgstr "Pasta de Destino (Inválida)"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/PreferencesDialog.cs:161
+#: NickvisionTubeConverter.GNOME/Blueprints/preferences_dialog.blp:249
+#: NickvisionTubeConverter.GNOME/Blueprints/preferences_dialog.blp:263
+msgid "Select Cookies File"
+msgstr "Selecione Ficheiro de Cookies"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/AddDownloadDialog.cs:440
+#: NickvisionTubeConverter.GNOME/Blueprints/add_download_dialog.blp:129
+msgid "Select Save Folder"
+msgstr "Selecione Pasta de Destino"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/MainWindow.cs:607
+msgid ""
+"Some downloads are still in progress.\n"
+"Are you sure you want to close Tube Converter and stop the running downloads?"
+msgstr ""
+"Algumas transferências ainda estão a decorrer.\n"
+"Tem a certeza que quer fechar o Tube Converter e parar as transferências em "
+"curso?"
+
+#: ../../../Models/DownloadManager.cs:167
+msgid "Some downloads finished with errors!"
+msgstr "Algumas transferências terminaram com erros!"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Controls/DownloadRow.cs:219
+msgid "Stopped"
+msgstr "Parado"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Controls/DownloadRow.cs:202
+msgid "Success"
+msgstr "Sucesso"
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/MainWindow.cs:781
+msgid ""
+"The authors of Nickvision Tube Converter are not responsible/liable for any "
+"misuse of this program that may violate local copyright/DMCA laws. Users use "
+"this application at their own risk."
+msgstr ""
+"Os autores do Nickvision Tube Converter não são responsáveis ​​por qualquer "
+"uso indevido deste programa que possa violar as leis locais de direitos de "
+"autor/DMCA. Os utilizadores usam esta aplicação por sua própria conta e "
+"risco."
+
+#: ../../../../NickvisionTubeConverter.GNOME/Views/MainWindow.cs:790
+msgid "translator-credits"
+msgstr "translator-credits"
```

Comment:
this string is meant to place translator name an link (`NAME WEBSITE` or `NAME <EMAIL>`). It's `translator-credits` in English because this way its not shown when running the app with English locale.